src/shiftml/ase/calculator.py

```diff
@@ -0,0 +1,143 @@
+import os
+import urllib.request
+
+from metatensor.torch.atomistic import ModelOutput
+from metatensor.torch.atomistic.ase_calculator import MetatensorCalculator
+
+url_resolve = {
+    "ShiftML1.0": "https://tinyurl.com/3xwec68f",
+    "ShiftML1.1": "https://tinyurl.com/53ymkhvd",
```

Comment:
so you re-created the exact previous models using the new infrastructure? Should these have a different version (like `ShiftML1.0+metatensor`) just for clarity or are you sure this will produce the same outputs?

Reply:
I will discuss this with everyone involved - but in principle I agree, that there should be a ShiftML1.0rev or something that makes it clear.

Reply:
I will open an issue to remind me of that.